templates/orders/commission-meta-box-html.php

```diff
@@ -0,0 +1,203 @@
+                                ?>
+                            </div>
+                        </td>
+                        <td width="1%">
+                            <div class="view">
+                                <?php
+                                $amount = $item->get_total();
+                                $original_commission = $commission_data->get_admin_commission();
+
+                                if ( $order->get_total_refunded_for_item( $item_id ) ) {
+                                    $commission_refunded = ( $order->get_total_refunded_for_item( $item_id ) / $amount ) * $original_commission;
+                                }
+
+                                ?>
+                                    <bdi><?php echo wc_price( $original_commission, array( 'currency' => $order->get_currency() ) ); ?></bdi>
+                                <?php
+
+                                if ( $order->get_total_refunded_for_item( $item_id ) ) :
+                                    ?>
+                                        <small class="refunded"><?php echo wc_price( $commission_refunded, array( 'currency' => $order->get_currency() ) ); ?></small>
+                                    <?php
+                                endif;
+                                ?>
+                            </div>
+                        </td>
+                    </tr>
+                <?php endforeach; ?>
+                </tbody>
+            </table>
+        </div>
+        <div class="wc-order-data-row wc-order-totals-items wc-order-items-editable">
+            <table class="wc-order-totals">
+                <tbody>
+                <tr>
+                    <td class="label"><?php esc_html_e( 'Net total:', 'dokan-lite' ); ?></td>
+                    <td width="1%"></td>
+                    <td class="total">
+                        <?php echo wc_price( $order_total, array( 'currency' => $order->get_currency() ) ); ?>
+                    </td>
+                </tr>
+                <tr>
+                    <td class="label"><?php esc_html_e( 'Vendor earning:', 'dokan-lite' ); ?></td>
+                    <td width="1%"></td>
+                    <td class="total">
+                        <?php echo wc_price( $net_amount, array( 'currency' => $order->get_currency() ) ); ?>
+                    </td>
+                </tr>
+                </tbody>
+            </table>
+
+            <div class="clear"></div>
+
+
+            <table class="wc-order-totals" style="border-top: 1px solid #999; border-bottom: none; margin-top:12px; padding-top:12px">
+                <tbody>
+                <tr>
+                    <td class="label label-highlight"><?php esc_html_e( 'Total commission:', 'dokan-lite' ); ?></td>
+                    <td width="1%"></td>
+                    <td class="total">
+                        <?php echo wc_price( $total_commission, array( 'currency' => $order->get_currency() ) ); ?>
```

Comment:
why are we passing currency all the time? isn't it handled by WC?

Reply:
Vai i have done it as woocommerce.